### ISSUE TITLE
Bedrock Login Problem

### ISSUE THREAD

**Issue by SussyBob** [ViaVersion/ViaFabricPlus#614]
### General Information

![image](https://github.com/user-attachments/assets/884fa97a-3671-49ef-96d0-f673e01e222e)

VFP 3.4.9/1.21.1/Quilt Fabric Loader 11.0.0-alpha.3+0.102.0-1.21

### Bug Description

Unable to login into the microsoft account for minecraft bedrock server login, i tried a lot of times. No crash, but it always shows that.. my network is good enough tho..

### Steps to Reproduce

1. Open the Via Fabric Plus settings
2. Bedrock > Click to Set Account for Bedrock Edition
3. Link automatically opened
4. Proceed to do the things
5. It shows that (from the picture that i given)

### Expected Behavior

-

**Comment by FlorianMichael** [comment]
You need to click on "accept" when the login process asks you to.

**Comment by SussyBob** [comment]
where? the same thing instantly shows after this down below are completed:

![image](https://github.com/user-attachments/assets/aa5558a5-b2b7-4748-b433-4e65977ca683)

**Comment by SussyBob** [comment]
Just email without password, logon using the "one time use code" sent to my email.

**Comment by RaphiMC** [comment]
E-Mail code is not supported. See https://github.com/RaphiMC/MinecraftAuth/issues/23 and https://github.com/PrismLauncher/PrismLauncher/issues/1223#issuecomment-1806756846

TL;DR: Noone has figured out how to make E-Mail code login work, so its generally not supported. Switch to password based authentication.
